src/property.ts

```diff
@@ -0,0 +1,125 @@
+import { BaseProperty } from 'admin-bro'
+
+const ID_PROPERTY = '_id'
+const VERSION_KEY_PROPERTY = '__v'
+
+class Property extends BaseProperty {
+    // TODO: Fix typings
```

Comment:
@JonaszJestem you can create a task in the backlog to do this so we won't forget